frontend/src/pages/Settings/settingsArr.css

```
@@ -2103,11 +2137,91 @@
+  .settings-arr .arr-btn--icon,
+  .arr-portal .arr-btn--icon {
+    width: 2.75rem;
+    min-width: 2.75rem;
+    height: 2.75rem;
+    min-height: 2.75rem;
+  }
+
+  .settings-arr .pill-toggle {
+    --w: 2rem;
+    --h: calc(var(--w) * 0.4865);
+    width: 2.75rem;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)